build/int.cloudbuild.yaml

```diff
@@ -228,7 +228,28 @@ steps:
     - verify-mig-with-percent-simple-local
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do destroy mig-with-percent-simple-local']
-
+- id: go-init-statful-mig
+  waitFor:
+    - create-all
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['/bin/bash', '-c', 'cd test/integration && RUN_STAGE=init go test -v -run TestMigStatefulModule ./... -p 1']
```

Comment:
This can be simplified as:
`cft test run TestMigStatefulModule --stage apply --verbose`

And all throughout...

Reply:
Thanks. Updating it